controllers/v1/cloud-services/aws.js

```diff
@@ -179,5 +182,80 @@ module.exports = class Aws {
     })
    }
 
+        /**
+     * @api {post} /kendra/api/v1/cloud-services/aws/uploadToPresignedUrl  
+     * upload file to presignedUrl
+     * @apiVersion 1.0.0
+     * @apiGroup Aws
+     * @apiHeader {String} X-authenticated-user-token Authenticity token
+     * @apiParamExample {json} Request:
+     * {
+     * "presignedUrl" : "https://storage.googleapis.com/sl-dev-storage/qrcode/R2I3A5/R2I3A5.png?GoogleAccessId=sl-dev-storage%40shikshalokam.iam.gserviceaccount.com&Expires=1587437414&Signature=N7yg1ixFXPG14vOxw7zPxMSm98qDoeY7IYtB8iRnSQRYTNwWRfAT7MqvLT3HW9iXexABEhhWPEDhVQt0J42BzY1it3mFMV1C9xgCl8Q%2BhuMVY746GIdapGEJEhicMGIeEVRy%2FItfuNg9UxcZyk1M3TU%2FEabQLCjAgMthnUvQ8tCqiH%2B1t%2FcJDODAvLv96sQVbO%2Fg4aW%2Bz3GGmtx39Kq%2FRTWVgxLBWu5wPs5PuDq7Xg6HSiD9peQylb7wRStkkuRP%2FMyjIhOuQTtgzPdNFV26I5WZu2Eu2EM5Hx6vDAYjXTOHuOry8fX6od5gcHMSjo9J645nNcN8tb97BIuG%2BgKOcg%3D%3D"
+     * "fileName":"filename.csv"
+     * }
+     * @apiSampleRequest /kendra/api/v1/cloud-services/aws/uploadToPresignedUrl
+     * @apiUse successBody
+     * @apiUse errorBody
+     * @apiParamExample {json} Response:
+     * {
+     * "message": "File Uploaded successfully",
+     * "status": 200,
+     * }
+     */
+
+    /**
+      * To upload file cloud 
+      * @method
+      * @name upload
+      * @param  {Request}  req  request body.
+      * @param  {files}  req.files.file -actual file to upload
+      * @param  {String}  req.body.presignedUrl -cloud url to upload
+      * @param  {String}  req.body.fileName - name of file
+      * @returns {JSON} Response with status and message.
+    */
+   async uploadToPresignedUrl(req) {
```

Comment:
@rakeshSgr - Create one upload endpoint that internally uploads it to google cloud storage. Also make it secure by internal access token

Reply:
resolved